src/docs/product/performance/index.mdx

```diff
@@ -28,7 +28,7 @@ Using the information on this page, you can trace issues back through services (
 
 <Note>
 
-For organizations participating in the limited release of [metrics](/product/sentry-basics/metrics/), the transaction filter on the Performance page has been simplified. The search bar can only be used to filter by transaction name, rather than to search for transactions matching the query. See [transaction search](/product/performance/filters-display/#filtering-transactions).
+For organizations participating in the limited release of [dynamic sampling](/product/sentry-basics/sampling/), the transaction filter on the Performance page has been simplified. The search bar can only be used to filter by transaction name, rather than to search for transactions matching the query. See [transaction search](/product/performance/filters-display/#filtering-transactions).
```

Comment:
```suggestion
For organizations participating in the Early Adopter program, the transaction filter on the Performance page has been simplified. The search bar can only be used to filter by transaction name, rather than to search for transactions matching the query. See [transaction search](/product/performance/filters-display/#filtering-transactions).
```